master/internal/rm/kubernetesrm/pods.go

```diff
@@ -1568,7 +1568,11 @@ func (p *pods) getNonDetPods() []k8sV1.Pod {
 		return nonDetPods
 	}
 	for _, p := range pList.Items {
-		if _, ok := p.Labels["determined"]; !ok {
+		_, ok1 := p.Labels[determinedLabel]
+		_, ok2 := p.Labels[determinedSystemLabel]
+		_, ok3 := p.Labels[determinedPreemptionLabel]
```

Comment:
I don't understand why we're filtering the determined preemption label, too. I think anything with that label will definitely have the determined label.

Reply:
I more saw it as a "why not" check, but if adding the check for `determinedPreemptionLabel` could cause confusion by inadvertently implying that pods can contain this label without having the `determinedLabel`, then I agree it makes sense to remove it.